### PR TITLE
Issue 633 about volpiano popup

### PR DESCRIPTION
**Pull request by dchiller**
This PR partially addresses #633 by creating a pop-up modal for additional information about Volpiano search on Cantus Ultimus. Before that issue is closed, the contents of the pop-up box should be reviewed and revised. 

Some features of this request:
- Creates a modal box that is accessed by a link that appears whenever Volpiano or Volpiano (Literal) search types are selected.
- Adjusts modal view to allow for multiple modals on the same page (Note: Our framework continues to only support opening a single modal at a time).
- Makes visible the "x" button to close all modals.
- Reopens a Search modal when the About Volpiano modal is closed if the search interface was in a modal (ie. if search is accessed from the main navbar rather than the manuscript detail view).
- Provides a draft of some "About Volpiano" text, including a review of important aspects of the way volpiano search is implemented in Cantus Ultimus.

### CODE REVIEW

File: nginx/public/node/frontend/public/js/app/search/chant-search/AboutVolpianoView.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import _ from "underscore";
```
Comment (jacobdgm) [inline nginx/public/node/frontend/public/js/app/search/chant-search/AboutVolpianoView.js:1]: What is going on here?

File: nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html [hunk 2]
```diff
@@ -1 +1,90 @@
...
+<div class="container-fluid">
+    <div class="row">
+        <p>
+            Volpiano is a font that was created for the transcription of chant
+            melodies. It was developed Fabian Weber at the University of
+            Regensburg under the direction of David Hiley.
+        </p>
+        <p>
+            With Volpiano, alphanumeric characters and other symbols are
+            rendered as pitches and other musical elements on standard staffs.
```
Comment (jacobdgm) [inline nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html:10]: the pedant in me is impelled to assert that "staves" is the plural of "staff". But "staffs" is probably clearer than "staves" in this situation.

File: nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html [hunk 3]
```diff
@@ -1 +1,90 @@
...
+    </div>
+    <div class="row">
+        <p><b>Features of Volpiano Search</b></p>
+        <ol>
+            <li>
+                There are two types of Volpiano search in Cantus Ultimus:
+                "Volpiano" and "Volpiano (Literal)." "Volpiano (Literal)"
+                matches the way pitches in the query are divided into neumes. In
+                Volpiano, a hyphen ("-") separates pitches belonging to
+                different neumes. For example, the two queries below will yield
```
Comment (jacobdgm) [inline nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html:51]: "the two queries below": I'm only seeing one query, `1---fgfh---33---fg-fh--` ...?

**Review by jacobdgm — APPROVED** [review]
I haven't tested this locally, but nothing looks terribly amiss. Feel free to make changes based on my comments or to ignore them, as necessary.

File: nginx/public/node/frontend/public/js/app/search/chant-search/AboutVolpianoView.js (hunk 1 above)
Comment (dchiller) [reply nginx/public/node/frontend/public/js/app/search/chant-search/AboutVolpianoView.js:1]: Nope....a wayward and unedited copy-paste situation...thanks!

File: nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html (hunk 3 above)
Comment (dchiller) [reply nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html:51]: Good point, let me add them

**Comment by dchiller** [comment]
Adding some screenshots of how the "About Volpiano" pop-up looks with this merge:

image
image

**Pull request merged**